packages/metal-quartz-link/src/MetalQuartzLink.soy

```diff
@@ -0,0 +1,56 @@
+{namespace MetalQuartzLink}
+
+/**
+ * This renders an anchor element.
+ */
+{template .render}
+	{@param? dataToggle: string}
+	{@param? elementClasses: string}
+	{@param? href: string}
+	{@param? html: html}
+	{@param? icon: ?}
+	{@param? label: html}
+	{@param? id: string}
+
+	{if $html}
+		{$html}
```

Comment:
I think we need to provide this until we can find a way to provide custom attributes in soy. We will need to reimplement link in other components that might use attributes we haven't provided in this component.